android-exoplayer/src/main/java/com/brentvatne/exoplayer/VideoEventEmitter.java

```diff
@@ -206,6 +219,19 @@ void buffering(boolean isBuffering) {
         receiveEvent(EVENT_BUFFER, map);
     }
 
+    void playbackStateChanged(boolean isPlaying) {
+        WritableMap map = Arguments.createMap();
+        map.putBoolean(EVENT_PROP_IS_PLAYING, isPlaying);
```

Comment:
Not clear to me in terms of API ...
should we add an isPlaying event or should we use playback rate value update.
IsPlaying is more simple PlaybackRateChange is more flexible ...

Reply:
please also notice this ticket : https://github.com/react-native-video/react-native-video/issues/2684

Reply:
In terms of API I think this makes sense because it is similar to how ExoPlayer handles/names this change, in this case we're only limited to is playing or not, but this can be expanded in the future to support all [player states](https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/Player.State.html)

Reply:
No problem, ok for me, it's also part of mediasession PR